quickwit/quickwit-config/src/node_config/mod.rs

```diff
@@ -375,7 +376,12 @@ impl NodeConfig {
 
     #[cfg(any(test, feature = "testsuite"))]
     pub fn for_test() -> Self {
-        serialize::node_config_for_test()
+        serialize::node_config_for_test(false)
+    }
+
+    #[cfg(any(test, feature = "testsuite"))]
```

Comment:
We won't do a lot of test for the header, it's a bit overkill to add this method (see commit on my branch fmassot/es_header)

Reply:
I merged your PR